From: Debian GNOME Maintainers <pkg-gnome-maintainers@lists.alioth.debian.org>
Date: Tue, 27 Nov 2018 16:09:31 +0300
Subject: 0006-use-paper-size-default-in-etc-papersize

---
 cupshelpers/cupshelpers.py | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/cupshelpers/cupshelpers.py b/cupshelpers/cupshelpers.py
index 3e09c01..c4d66ad 100755
--- a/cupshelpers/cupshelpers.py
+++ b/cupshelpers/cupshelpers.py
@@ -722,6 +722,23 @@ def setPPDPageSize(ppd, language):
     for each in letter:
         if language == each:
             size = 'Letter'
+
+    # Use setting in /etc/papersize if available
+    try:
+        f = open ("/etc/papersize")
+        for line in f:
+            if line.startswith("#"):
+                continue
+            if line.strip().lower().startswith("a4"):
+                size = 'A4'
+            elif line.strip().lower().startswith("letter"):
+                size = 'Letter'
+            elif line.strip() != "":
+                break
+        f.close()
+    except:
+        pass
+
     try:
         ppd.markOption ('PageSize', size)
         _debugprint ("set PageSize = %s" % size)
